### ISSUE TITLE
Running development environment

### ISSUE THREAD

**Issue by Bost** [ExpDev07/coronavirus-tracker-api#183]
When running the service locally, I get:
```fish
> flask run
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
```
Then in the browser, after opening:
-  `http://127.0.0.1:5000/`I get redirected to `https://github.com/ExpDev07/coronavirus-tracker-api`
-  `http://127.0.0.1:5000/all`I get to see the result of the `v1` version.

So far everything's fine[1]. But how do I get to see the results for the `v2` version? So far I can see only the `v1` routes are activated:

```fish
> /usr/bin/rg --smart-case api.route
app/routes/v1/recovered.py
5:@api.route('/recovered')

app/routes/v1/deaths.py
5:@api.route('/deaths')

app/routes/v1/all.py
5:@api.route('/all')

app/routes/v1/confirmed.py
5:@api.route('/confirmed')
```
And can I ask you such things somewhere on some channel, discussion forum, chat, etc.? Without opening an issue here? Thanks

[1] the redirect is rather a nuisance

**Comment by ExpDev07** [comment]
We've moved over to FastAPI and are using Flask only for the version 1 routes. You need to start the app by running ``pipenv run dev`` for development (or ``pipenv run start`` if you don't want live-reload when you make changes to files). But make sure to run ``pipenv install`` first so that you get the new dependencies.

You can add me on Discord Marius Big Ounce#2997 if you have any questions.

**Comment by Bost** [comment]
HA!
```
> git clone https://github.com/ExpDev07/coronavirus-tracker-api.git
Cloning into 'coronavirus-tracker-api'...
remote: Enumerating objects: 166, done.
remote: Counting objects: 100% (166/166), done.
remote: Compressing objects: 100% (142/142), done.
remote: Total 1349 (delta 98), reused 79 (delta 23), pack-reused 1183
Receiving objects: 100% (1349/1349), 295.57 KiB | 1.10 MiB/s, done.
Resolving deltas: 100% (769/769), done.
> cd coronavirus-tracker-api
[20:25:16] bost@bost-new-64 /tmp/coronavirus-tracker-api (0) 
> pipenv install
Creating a virtualenv for this project…
Using /usr/local/bin/python3.8 (3.8.2) to create virtualenv…
⠋Running virtualenv with interpreter /usr/local/bin/python3.8
Using base prefix '/usr/local'
/usr/lib/python3/dist-packages/virtualenv.py:1090: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import imp
New python executable in /home/bost/.local/share/virtualenvs/coronavirus-tracker-api-UDBrV3nM/bin/python3.8
Also creating executable in /home/bost/.local/share/virtualenvs/coronavirus-tracker-api-UDBrV3nM/bin/python
Installing setuptools, pkg_resources, pip, wheel...done.

Virtualenv location: /home/bost/.local/share/virtualenvs/coronavirus-tracker-api-UDBrV3nM
Installing dependencies from Pipfile.lock (04376b)…
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 25/25 — 00:00:03
To activate this project's virtualenv, run the following:
 $ pipenv shell
[20:25:32] bost@bost-new-64 /tmp/coronavirus-tracker-api (0) 
> pipenv shell
Spawning environment shell (/usr/bin/fish). Use 'exit' to leave.
[20:25:40] bost@bost-new-64 /tmp/coronavirus-tracker-api (0) 
> source /home/bost/.local/share/virtualenvs/coronavirus-tracker-api-UDBrV3nM/bin/activate.fish
(coronavirus-tracker-api-UDBrV3nM) [20:25:53] bost@bost-new-64 /tmp/coronavirus-tracker-api (0) 
> pipenv run dev
Traceback (most recent call last):
  File "/usr/bin/pipenv", line 11, in <module>
    load_entry_point('pipenv==11.9.0', 'console_scripts', 'pipenv')()
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/pipenv/vendor/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pipenv/cli.py", line 602, in run
    core.do_run(command=command, args=args, three=three, python=python)
  File "/usr/lib/python3/dist-packages/pipenv/core.py", line 2200, in do_run
    command = ' '.join(project.scripts[command])
  File "/usr/lib/python3/dist-packages/pipenv/project.py", line 374, in scripts
    scripts[k] = shlex.split(v, posix=True)
  File "/usr/lib/python3.7/shlex.py", line 309, in split
    return list(lex)
  File "/usr/lib/python3.7/shlex.py", line 299, in __next__
    token = self.get_token()
  File "/usr/lib/python3.7/shlex.py", line 109, in get_token
    raw = self.read_token()
  File "/usr/lib/python3.7/shlex.py", line 140, in read_token
    nextchar = self.instream.read(1)
AttributeError: 'list' object has no attribute 'read'
```
Anyway it would be fine to put such instructions to the README.md

**Comment by ExpDev07** [comment]
I think you need to update your python version to 3,8, also, I have updated the README with those instructions (see Running / Development). I'm on 
```
$ python --version
Python 3.8.1
```

And it works just fine.

**Comment by Bost** [comment]
Hmm :)
```fish
> python3 --version
Python 3.8.2
```

**Comment by ExpDev07** [comment]
@Bost 

Weird. You're ``pipenv`` is telling you ``File "/usr/lib/python3.7/shlex.py", line 309, in split`` (python 3.7).

**Comment by Bost** [comment]
The problem persists. I tried different machines (both of them ubuntu 19.10)... my python 3.8.2 is compiled from the source on both of them... I use the fish-shell so just to rule out any interference with that I also created myself a new user having bash... I also tried to run `pipenv --python 3.8 ...`. Nothing works... I'm stuck. Like a thumb in... wherever.

I've been having issues with the python environment configuration (python2 vs python3 vs updates) nightmare already in the past. On different occasions on different machines... And now it's back :(

I'm calling it a day.
